File: src/main/java/no/nav/syfo/api/intern/domain/RSPerson.java
package no.nav.syfo.api.intern.domain;

import lombok.Data;
import lombok.experimental.Accessors;

import java.time.LocalDateTime;

@Data
@Accessors(fluent = true)
public class RSPerson {
    public String navn;
    public String aktoerId;
    public String fnr;
    public String epost;
    public String tlf;
    public LocalDateTime sistInnlogget;
    public Boolean samtykke;
}

File: src/main/kotlin/no/nav/syfo/api/v3/domain/oppfoelgingsdialog/RSPerson.kt
package no.nav.syfo.api.v3.domain.oppfoelgingsdialog

import java.time.LocalDateTime

data class RSPerson(
    var navn: String? = null,
    var aktoerId: String? = null,
    var fnr: String? = null,
    var epost: String? = null,
    var tlf: String? = null,
    var sistInnlogget: LocalDateTime? = null,
    var samtykke: Boolean? = null
)
